src/Orchard.Web/Modules/Orchard.DynamicForms/Services/IFormService.cs

```diff
@@ -20,6 +21,7 @@ public interface IFormService : IDependency {
         Submission CreateSubmission(Submission submission);
         Submission GetSubmission(int id);
         IPageOfItems<Submission> GetSubmissions(string formName = null, int? skip = null, int? take = null);
+        Stream ExportSubmissions(string formName = null);
```

Comment:
Breaking change. Please create a new interface `IFormExportService` and have an `ExportXml` in it. Or target `dev`.

Reply:
As discussed on the meeting, we can break stuff now. :D